shoes-core/spec/shoes/common/ui_element_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Shoes::Common::UIElement do
+  include_context "dsl app"
+
+  subject { test_class.new }
+
+  let(:test_class) do
+    Class.new do
+      include Shoes::Common::UIElement
+
+      # Override UIElement's Initialization include for simpler testing
+      def initialize
+      end
+    end
+  end
+
+  describe "stubbed updates" do
+    it { is_expected.to respond_to(:update_fill) }
+    it { is_expected.to respond_to(:update_stroke) }
+  end
+
+  describe "#needs_rotate?" do
+    before do
+      # Add faux rotate accessor to prove it really doesn't support rotation
+      class << subject
+        attr_accessor :rotate
+      end
+    end
```

Comment:
minor thing but I feel like this is overdoing it, couldn't we just stub the `rotate` method to have the same effect with less code and more in line with the rest of the code base or am I missing something? :)

If we use the instance_variable we could revert to using the method or use instance_variable_set imo